pom.xml

```diff
@@ -216,7 +216,7 @@
         <dependency>
             <groupId>com.github.thebusybiscuit</groupId>
             <artifactId>CS-CoreLib2</artifactId>
-            <version>0.12</version>
+            <version>0b974b0</version>
```

Comment:
I don't know why, but 0.13 just didn't work. Seems like the repo is not up to date

Reply:
0.13 works fine for me and also for our GitHub CI. Maybe delete your .m2 maven cache and try to build it again.

Reply:
I mean, its not compatible with minecraft version 1.13.2

Reply:
Oh that is fine.
Backwards compatibility with 1.13 is a bit broken in current versions, that will be fixed.
You don't have to worry about that right now though, since it is a different project.
Simply build against the latest version.